QUICKSTART.md

```diff
@@ -6,7 +6,7 @@ Step 0:
 Create a Google cloud project and a VM running 'Ubuntu 18.04 LTS Minimal' in the [Google console](https://console.cloud.google.com/compute/instances).
 
 You'll also need to create a [firewall rule](https://console.cloud.google.com/networking/firewalls/list) to allow
-traffic on ports 8000 and 8080, and add that rule to your VM network tags.
+traffic on ports 8000, and add that rule to your VM network tags.
```

Comment:
I think 8080 is more standard, and would then work without config with more environments.